en/docs/sdks/sdk-overview.md

```diff
@@ -25,13 +25,14 @@ Apart from the above guides provided, you can use the following Github repositor
 
 - [.NET OIDC SDK](https://github.com/asgardeo/asgardeo-dotnet-oidc-sdk)
 
-- [React Native OIDC SDK](https://github.com/asgardeo/asgardeo-react-native-oidc-sdk)
-
-- [Auth Single Page Application SDK](https://github.com/asgardeo/asgardeo-auth-spa-sdk)
-
-- [Spring-boot application](https://github.com/wso2-extensions/identity-samples-spring-boot)
+<!-- - [React Native OIDC SDK](https://github.com/asgardeo/asgardeo-react-native-oidc-sdk) -->
 
+- [Auth React SDK](https://github.com/asgardeo/asgardeo-auth-react-sdk)
```

Comment:
How are these different from the ones at the top? We have a guides in the doc space on JavaScript, Angular, and React.